src/reloader.coffee

```diff
@@ -88,12 +88,17 @@ exports.Reloader = class Reloader
       if path.match(/\.(jpe?g|png|gif)$/i)
         @reloadImages(path)
         return
+    if options.isChromeExtension
+      @reloadChromeExtension
```

Comment:
Hey, does this actually work? This line ain't calling that function, it's missing `()` in the end.